android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,11 +1,11 @@
-arguments=
+arguments=--init-script /home/undefined/.config/Code/User/globalStorage/redhat.java/1.16.0/config_linux/org.eclipse.osgi/51/0/.cp/gradle/init/init.gradle --init-script /home/undefined/.config/Code/User/globalStorage/redhat.java/1.16.0/config_linux/org.eclipse.osgi/51/0/.cp/gradle/protobuf/init.gradle
 auto.sync=false
 build.scans.enabled=false
-connection.gradle.distribution=GRADLE_DISTRIBUTION(VERSION(6.0))
+connection.gradle.distribution=GRADLE_DISTRIBUTION(VERSION(7.4.2))
 connection.project.dir=
 eclipse.preferences.version=1
 gradle.user.home=
-java.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home
+java.home=/usr/lib/jvm/java-11-openjdk-amd64
 jvm.arguments=
 offline.mode=false
 override.workspace.settings=true
```

Comment:
These files were generated when I did the initial project setup. I feel we can get rid of them and add their path to `.gitignore`